lib/order.rb

```diff
@@ -0,0 +1,48 @@
+require 'csv'
+# require 'awesome_print'
+
+require_relative 'customer'
+
+#helper method
+def string_to_hash(string)
+  hash = string.split(/;/).map { |pair| pair.split(/:/) }.to_h
+  return hash.each { |key, value| hash[key] = value.to_f if value =~ /\d+/ }
+end
```

Comment:
This is a great helper method.  However I suggest making it a private class method.